src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -166,6 +166,10 @@ The `type` and default key is `"os"`.
 
 : _Optional_. A flag indicating whether the OS has been jailbroken or rooted.
 
+`brightness`
+
+: _Optional_. Either `light` or `dark`. Describes whether the OS runs in dark mode or not.
```

Comment:
wondering if this should be OS only, can I have an OS with light mode but my App is Dark mode?

Reply:
That's why I said that this field also makes sense in the App context.
Though the at the app level the SDK can't really detect it and the user would have to set it himself.

A lot of apps have options for forcing it to be light/dark or to have it detect the OS mode. Like Twitter for example.

Reply:
oh just noticed its in the description, my bad :P